python-backend/app/commands.py

```diff
@@ -22,7 +22,9 @@
 from .api.mines.status.models.mine_operation_status_sub_reason_code import MineOperationStatusSubReasonCode
 from .api.utils.random import generate_mine_no, generate_mine_name, random_geo, random_key_gen, random_date, random_region, random_mine_category
 from .api.parties.party_appt.models.mine_party_appt import MinePartyAppointment
-from .extensions import db
+from .extensions import db, sched
+from .scheduled_jobs import NRIS_jobs
```

Comment:
Unused import?

Reply:
The function being called by the CLI command lives in the NRIS_jobs file.